scripts/migration/migrate_registration_extra.py

```diff
@@ -16,15 +17,38 @@
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 
+def migrate_file_representation(bad_file):
+    view_url = bad_file.get('viewUrl', '')
+    fid = view_url.split('/')[-2]
+    f = FileNode.load(fid)
+    data = {
+        'data': {
+            'kind': 'file',
+            'name': bad_file['selectedFileName'],
+            'path': f.path,
+            'extra': {},
+            'sha256': f.get_version().metadata['sha256']
+        }
+    }
+    bad_file.update(data)
+    logger.info('Migrated file representation of File: {0}'.format(fid))
```

Comment:
Better to log *before* making the update to bad_file, e.g. `Migrating file representation...`. That way, if the script crashes on any of the update steps, we can diagnose the `fid` that failed.